app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Users::ConfirmationsController < Devise::ConfirmationsController
```

Comment:
Should we just delete the dead code? Dont see why we need to keep it (unless this is something we may need to overwrite later?)

Reply:
Yeah, I kept it for the future in case we need to overwrite something else. It also gives some information about how the devise is working so I thought keeping them for a while might be useful at least until we are completely done with the authentication.